config/application.config.php

```diff
@@ -1,22 +1,29 @@
 <?php
-return [
-    // This should be an array of module namespaces used in the application.
-    'modules' => [
-        'ZendDeveloperTools',
-        'DoctrineModule',
-        'DoctrineORMModule',
-        'User',
-        'Application',
-        'Company',
-        'Decision',
-        'Frontpage',
-        'Activity',
-        'Photo',
-        'Education',
-        'Migration',
-        'MaglMarkdown'
-    ],
+$env = getenv('APP_ENV') ?: 'production';
```

Comment:
I think that it is bad to include such code in a config file. It would be better to distuinguish between both in the `index.php`.